### PR TITLE
Add find defintions for elisp-mode

### PR DESCRIPTION
**Pull request by Sasanidas**

### CODE REVIEW

**Comment by cxxxr** [comment]
@Sasanidas 
Is this still in process?

**Comment by Sasanidas** [comment]
> @Sasanidas Is this still in process?

Oh sorry, I think this is ready for review :+1:

**Comment by cxxxr** [comment]
Looks good.
Thank you!

File: modes/elisp-mode/lem-elisp-mode.asd [hunk 1]
```diff
@@ -2,6 +2,7 @@
   :depends-on ("lem"
                "jsonrpc"
                "lem-lisp-mode"
+               "jsonrpc/transport/http"
```
Comment (cxxxr) [inline modes/elisp-mode/lem-elisp-mode.asd:5]: @Sasanidas 
I'm sorry.
After all, this dependence on ironclad, crack, etc. seems to be quite large.
Is there any way we can delay it until we use a specific feature of elisp-mode?

Comment (Sasanidas) [reply modes/elisp-mode/lem-elisp-mode.asd:5]: Umm I see, I can separate the elisp-mode form the rpc interface, so elisp-mode will only provide basic major-mode

Comment (Sasanidas) [reply modes/elisp-mode/lem-elisp-mode.asd:5]: I think this should solve the issue https://github.com/lem-project/lem/pull/825 :+1:

**Pull request merged**